src/main/java/seedu/address/logic/commands/ListProcCommand.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_CLIENT_PROCEDURES;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.client.Client;
+
+public class ListProcCommand extends Command {
+
+    public static final String COMMAND_WORD = "listProc";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Lists the procedures of a client in the index. "
+            + "Parameters: INDEX (must be a positive integer) \n"
+            + "Example: listProc 1";
+
+    public static final String MESSAGE_SUCCESS = "Procedures successfully loaded.";
+
```

Comment:
Just a suggestion. Perhaps we can add more feedback if there is no Procedures?

`public static final String MESSAGE_FAIL = "No Procedures to load!";`